sdk/core/core-util/src/bytesEncoding.ts

```diff
@@ -1,25 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-/** The supported character encoding type */
```

Comment:
Where do these come from now?  I would have expected the same pattern of `export function myFunc() { return tsp.myFunc() }` everywhere so just wanted to better understand - maybe they moved to `index` and I just haven't gotten there yet?

Reply:
Yeah, I moved everything that is just a redefinition into index -- felt a bit unnecessary having everything split over so many files if they're effectively just a re-export.